src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -41,6 +41,16 @@
   }
 }
 
+.HeroRecommendation-image-loading {
+  @include respond-to(medium) {
+    width: 40%;
+  }
+
+  .LoadingText {
+    height: 14em;
```

Comment:
This looks good, but I wonder where the value of `14em` comes from? What happens if you use `100%` instead?

Reply:
`100%` works! `em` was the unit used elsewhere for the height and I was trying to stay consistent. (`14em` is the approximate height of the image when the screen is at 1024px width). I have pushed through an updated stylesheet.

https://user-images.githubusercontent.com/28129806/132370694-f656d717-08d3-41f3-8852-0f837c084e4f.mov